README.md

```diff
@@ -24,11 +24,12 @@ cloud.gov functionality.
 
 ## Quickstart Demo
 
-This is a demo of the [Java application demo](https://cloud.gov/quickstart/).
+This is a demo of launching a [GoLang sample app](https://github.com/pburkholder/sample-app)
```

Comment:
Small quibble: The language is just called "Go", with some people using "golang" for search friendliness or hashtags. https://go.dev/doc/faq#go_or_golang